english/TheTaleOfTheProgrammer

```diff
@@ -0,0 +1 @@
+once upon a time there was a random 14 year old kid who decided to learn git and github. little did he know that the pulling request bullshit would fuck his brain.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.